securedrop/journalist_templates/base.html

```diff
@@ -17,6 +17,8 @@
     <div id="logout">
       {% if g.user and g.user.is_admin %}
       <a href="{{ url_for('admin_index') }}">Admin</a> | 
+      {% else %}
+      <a href="{{ url_for('edit_account') }}">Edit Account</a>
```

Comment:
Also, do we want to show "Edit account" on admin's pages too? They can always edit their own info through the Admin Interface, but they might appreciate a shortcut to their own settings via "Edit account" as well.